index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="ja">
+
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>Weather</title>
+	<link rel="stylesheet" href="stylesheet.css">
+</head>
+
+<body>
+	<div class="container">
+		<h1>Weather☺︎</h1>
+	</div>
+	<div class="city">
+		<label for="city">都市を選択して下さい</label>
+		<select name=region id="city" onchange="selectCity()" multiple>
```

Comment:
```suggestion
      <select name="region" id="city" onchange="selectCity()">
```
都市は複数選択できる必要ないと思うのでこうしましょう